### ISSUE TITLE
Operations processes

### ISSUE THREAD

**Issue by iteles** [dwyl/process-handbook#35]
**As a** dwyler,
**I want** to ensure that all operations processes are captured in a clear and replicable way, with checklists of the things that need to be done in a step-by-step fashion where possible
**So that** if the operations people are unavailable, the processes can still be carried out and in future we can automate these in order to free up time for other productive tasks!

1. Create a list of the various tasks that are performed
2. Open issues for each of these tasks in this repo 
3. Link each as a checklist in this main comment
4. Follow our [contributing process](https://github.com/dwyl/contributing) to complete these (I'm happy to be the reviewer)

**Comment by iteles** [comment]
@markwilliamfirth As discussed, this (and its counterpart for non-ops processes #36) are our top priority for dwyl summer 🎉

**Comment by markwilliamfirth** [comment]
This is quite a big task and any operational handbook can always be further improved upon with additional detail added to it. It may be better to timebox the amount of time you want me to spend on this and then I can assign that time to the task

**Comment by markwilliamfirth** [comment]
dwyl
-

**Finance**
- Bookkeeping
- Expenses
- Payroll / PAYE
- Contractor Invoicing
- Grants & Tax Credits
- Corporation Tax Return
- Budgeting
- Forecasting
- VAT Return

**Human Resources**
- Onboarding
- Offboarding
- Policy
- Resourcing
- Hiring
- Employee proofs
- Culture
- Feedback
- Visas
- OKRs
- Complaints

**Legal Documents**
- Writing
- Ratification
- Filing
- Trade marks

**Company Administration**
- Companies House accounts
- Articles of Association
- Insurance
- Company structure

**Marketing**
- Twitter

**Events**
- dwyl Summer

**Project Management**
- Scrum
- Client Management

Sales
- Requirements gathering
- Providing quotes
- Closing

Focus Hub
-

**Finance**
- Reconciliations

Office Management
- Maintenance

**Events**
- Yoga

**Sales**
- Tours
- Negotiation
- Closing

Tenant Management
- Onboarding
- Offboarding
- Deposits
- Identity verification
- Key cutting
- Requests

**Marketing**
- Twitter
- Gumtree
- Facebook Ads
- HUBBLE

**Comment by iteles** [comment]
Agreed, let's make it the focus of dwylsummer as much as possible, so the 5 days of next week.

I would suggest we start with Finance and Legal and go on from there, but it would be useful to also prioritise these in terms of the order we want to write them in

**Comment by markwilliamfirth** [comment]
@iteles what about other tasks that are currently priorities and ongoing activities such as payroll etc? It's not possible to assign all 5 days

**Comment by iteles** [comment]
Agreed, we need to time estimate those first and remove them from the overall 5 days (as well as a morning for dwylsummer water sports).

We should then assess at the end of the week and review how much more time will be required. 
My intention is to have a more focussed period where you (hopefully both of us) can get as much down as possible and then use the knowledge to understand how much time we should be setting aside per week to get this done.

Doing a repetitive task without documenting it is shouting into the void and we need to do a lot less of that 😬 

![hamster-wheel](http://gph.is/2bddurr)

**Comment by markwilliamfirth** [comment]
@iteles checking that you are sure you want the handbook recorded in issues? Perhaps it would be better to write the handbook as a document instead?

**Comment by iteles** [comment]
I would like an issue for each section and an `.md` file PR'ed into this repo for each one of those please 👍

**Comment by Cleop** [comment]
@iteles - perhaps you could update progress on this issue as there was a proposed time set aside to work on this which may/may not have materialised...

**Comment by iteles** [comment]
There was a `finance.md` file which was pushed directly to `master` and reviewed in #46, but no effort was made on any of the other points.
